artifactory_test.go

```diff
@@ -4148,8 +4148,16 @@ func TestRefreshableTokens(t *testing.T) {
 	// Upload a file and assert the refreshable tokens were generated.
 	artifactoryCommandExecutor := tests.NewJfrogCli(execMain, "jfrog rt", "")
 	uploadedFiles := 1
-	uploadWithSpecificServerAndVerify(t, artifactoryCommandExecutor, tests.RtServerId, "testdata/a/a1.in", uploadedFiles)
-	curAccessToken, curRefreshToken := getTokensFromConfig(t, tests.RtServerId)
+	err := uploadWithSpecificServerAndVerify(t, artifactoryCommandExecutor, tests.RtServerId, "testdata/a/a1.in", uploadedFiles)
+	if err != nil {
+		assert.NoError(t, err)
```

Comment:
Can we move this assertion to be inside the ```uploadWithSpecificServerAndVerify``` function and by that shorten the code? (this function si called multiple times).

The same goes for the ```getTokensFromConfig```, ```setPasswordInConfig``` and ```setRefreshTokenInConfig``` functions.